app/queries/base_path_state.rb

```diff
@@ -0,0 +1,71 @@
+module Queries
+  module BasePathState
+    extend ArelHelpers
+
+    def self.conflict_with_draft_content(content_item_id, base_path)
```

Comment:
Yeah - I think they it can probably be just a single method that handles both cases. There was a more complex condition before with draft (checking if it clashed with live) but that didn't work out so I think I can simplify them.

Thanks for the review by the way 👍